### DISCUSSION TITLE
Has this plug in stopped being supported?

### DISCUSSION THREAD

**Discussion by bite-Paul-Le-May** [reloadware/reloadium#206] [General]
I loved using it for a year under license, but now https://reloadium.io/ seems to no longer be up?

**Comment by dkrystki** [comment]
Hi @bite-Paul-Le-May,
I appreciate your support.
Yes, unfortunately I had to stop maintaining Reloadium.
Damian

**Reply by danielmiester** [reply]
If you are no longer developing Reloadium, please consider  open-sourcing it, or at least publishing a final revision that makes it permanently pro.

**Reply by dkrystki** [reply]
Hi @danielmiester 
Reloadium Pro is already free and permanently enabled after the trial is ended.
I will consider opensourcing it.

**Reply by danielmiester** [reply]
I have been long past the trial :P
I just uninstalled/reinstalled the plugin and it seems to think it is a pro version now :)

Thank you for freeing it up. Too many projects get abandoned and left in limbo regarding licences for existing users.

**Reply by heisen273** [reply]
hey Damian @dkrystki , really appreciate your hard work in this awesome project. 
Any insights/updates on open-sourcing? I'd love to contribute to this amazing project and attempt to fix few small issues related to loading of a watched files.

**Comment by MarkoNovi** [comment]
yeah, it would be awesome if, with open-sourcing, we could make a VSCode extension.
